js/app/dashboard/components/UnassignedTasks.js

```diff
@@ -49,49 +38,40 @@ class StandaloneTasks extends React.Component {
         </Draggable>
       )
     })
-  }
-}
-
-const StandaloneTasksWithConnect = connect(
-  (state) => ({
-    selectedTasksLength: selectSelectedTasks(state).length,
-  })
-)(StandaloneTasks)
+  },
+  (prevProps, nextProps) => {
```

Comment:
I think you can remove this whole function, because it checks all possible props. React.memo would do the same out of the box, as I understand